core/storage/feedback/gae_models.py

```diff
@@ -179,21 +243,40 @@ def create(cls, exploration_id, thread_id, message_id):
 
     @classmethod
     def get(cls, exploration_id, thread_id, message_id, strict=True):
-        """Gets the FeedbackMessageModel entry for the given ID.
-
-        If the message id is valid and it is not marked as deleted, returns the
-        message instance. Otherwise:
-        - if strict is True, raises EntityNotFoundError
-        - if strict is False, returns None.
+        """Gets the FeedbackMessageModel entry for the given ID. Raises error if
+            no undeleted messages are found and strict == True.
+
+        Args:
+            exploration_id: str. ID of the exploration
+            thread_id: str. ID of the thread.
+            message_id: int. ID of the message.
+            strict: bool. Whether to raise an error if no FeedbackMessageModel
+                entry is found for message_id.
+
+        Returns:
+            None, if strict == False and no undeleted message with the given
+                message_id exists in the datastore. Otherwise the message
+                instance that corresponds to the given id.
+
+        Raises:
+            EntityNotFoundError: if strict==True and message id is not valid
+                and it is not marked as deleted.
         """
         instance_id = cls._generate_id(exploration_id, thread_id, message_id)
         return super(FeedbackMessageModel, cls).get(instance_id, strict=strict)
 
     @classmethod
     def get_messages(cls, exploration_id, thread_id):
-        """Returns an array of messages in the thread.
+        """Returns an array of messages in the thread within the default
```

Comment:
It should be "list".

Ah, then the concept of "size" should appear somewhere in the docstring. So, e.g., "Returns a list of messages in the given thread. The number of messages returned is capped by feconf.DEFAULT_QUERY_LIMIT."